spectre-meltdown-checker.sh

```diff
@@ -662,6 +662,10 @@ check_variant2()
 			dd if=/dev/cpu/0/msr of=/dev/null bs=8 count=1 skip=9 2>/dev/null
```

Comment:
Both the elif you added will only be checked if `/dev/cpu/0/msr` can be found, even if you don't use it, is there a reason ?